frontend/src/features/VesselSidebar/Tabs.tsx

```diff
@@ -0,0 +1,149 @@
+import styled from 'styled-components'
+
+import { COLORS } from '../../constants/constants'
+import { VesselSidebarTab } from '../../domain/entities/vessel/vessel'
+import { showVesselSidebarTab } from '../../domain/shared_slices/Vessel'
+import { useMainAppDispatch } from '../../hooks/useMainAppDispatch'
+import { useMainAppSelector } from '../../hooks/useMainAppSelector'
+import { ReactComponent as ReportingSVG } from '../icons/Icone_onglet_signalement.svg'
+import { ReactComponent as VMSSVG } from '../icons/Icone_VMS_fiche_navire.svg'
+import { ReactComponent as ControlsSVG } from '../icons/Picto_controles.svg'
+import { ReactComponent as VesselIDSVG } from '../icons/Picto_identite.svg'
+import { ReactComponent as FisheriesSVG } from '../icons/Picto_peche.svg'
+import { ReactComponent as SummarySVG } from '../icons/Picto_resume.svg'
+
+export function Tabs() {
+  const dispatch = useMainAppDispatch()
+  const { selectedVessel, vesselSidebarTab } = useMainAppSelector(state => state.vessel)
+  const isAdmin = useMainAppSelector(state => state.global.isAdmin)
+
+  return (
+    <TabList>
+      {isAdmin && (
+        <Tab
```

Comment:
We have the icons both as clean React components and as `IconButton` in Monitor UI.

If some icons are missing, it's easy to add them (just [run](https://github.com/MTES-MCT/monitor-ui/blob/main/package.json#L22) `yarn build:icons` once the [src/assets/icons](https://github.com/MTES-MCT/monitor-ui/tree/main/src/assets/icons) folder is updated.

Reply:
I added a TODO